presentation/valid/ttml2-prstn-shear-linear-mapping.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tt xml:lang="ja"
+    xmlns="http://www.w3.org/ns/ttml"
+    xmlns:ttm="http://www.w3.org/ns/ttml#metadata"
+    xmlns:ttp="http://www.w3.org/ns/ttml#parameter"
+    xmlns:tts="http://www.w3.org/ns/ttml#styling"
+    ttp:profile="ttml2-presentation"
+    xmlns:ttva="http://skynav.com/ns/ttv/annotations" ttva:warnOn="all" ttva:expectedErrors="0" ttva:expectedWarnings="0" ttva:model="ttml2">
+  <head>
+    <metadata>
+      <ttm:desc>#shear - linear mapping</ttm:desc>
+    </metadata>
+    <styling>
+      <initial tts:color="white"/>
+      <initial tts:fontSize="1.000001c"/>
+      <style xml:id="s1" tts:showBackground="whenActive" tts:backgroundColor="black" tts:textAlign="center" tts:displayAlign="center"/>
```

Comment:
Addressed in 4543e2c.

Reply:
This comment https://github.com/w3c/ttml2-tests/pull/248/files#r397799471 has not been addressed by 4543e2c since `presentation/valid/ttml2-prstn-shear-linear-mapping.xml` appears still to use `initial` values. That commit only addressed the issue on `presentation/valid/ttml2-prstn-line-shear-linear-mapping.xml`.

Reply:
@nigelmegitt please recheck, addressed in b861764

Reply:
Thanks, that's done it @skynavga .